.github/ISSUE_TEMPLATE/user-story.yml

```diff
@@ -0,0 +1,62 @@
+name: User Story
+description: Capture actionable sprint work
+title: "[Story]: "
```

Comment:
Like the bug template, not married to prefixing them, so we'll just be consistent whichever way we go. The prefix is not for search/organization purposes. The default label does that for us.